lib/matplotlib/tests/test_legend.py

```diff
@@ -1187,12 +1181,9 @@ def test_plot_single_input_multiple_label(label_array):
     x = [1, 2, 3]
     y = [2, 5, 6]
     fig, ax = plt.subplots()
-    with pytest.warns(mpl.MatplotlibDeprecationWarning,
-                      match='Passing label as a length 2 sequence'):
+    with pytest.raises(ValueError,
+                       match='label must be scalar or have the same length'):
         ax.plot(x, y, label=label_array)
-    leg = ax.legend()
-    assert len(leg.get_texts()) == 1
-    assert leg.get_texts()[0].get_text() == str(label_array)
```

Comment:
This test is parametrized over 3 types of sequence.  Should I change it to just test with one?

Reply:
I decided yes, since the parametrization was presumably because we cared what output string looked like, but we don't any more.